sensing/pointcloud_preprocessor/src/concatenate_data/concatenate_and_time_sync_nodelet.cpp

```diff
@@ -145,10 +145,10 @@ PointCloudConcatenateDataSynchronizerComponent::PointCloudConcatenateDataSynchro
 
   // Subscribers
   {
-    RCLCPP_INFO_STREAM(
```

Comment:
[imo]
This is just printed during the initialization process. 
Can we keep this as INFO?